src/components/Collections/TokenDetail/index.tsx

```diff
@@ -148,97 +171,142 @@ function TokenDetail({ contractAddress, tokenId }: TokenDetailProps) {
     return <NotFound />;
   }
 
+  const isOwner =
+    system.tzPublicKey &&
+    (system.tzPublicKey === token.owner ||
+      system.tzPublicKey === token.sale?.seller);
+
   return (
-    <Flex flex="1" width="100%" minHeight="0">
-      <Flex flexDir="column" w="50%" h="100%" overflowY="scroll">
-        <Flex py={8} px={8}>
-          <MinterButton
-            variant="primaryActionInverted"
-            onClick={e => {
-              e.preventDefault();
-              setLocation('/collections', { replace: true });
-            }}
-          >
-            <Box color="currentcolor">
-              <ChevronLeft size={16} strokeWidth="3" />
-            </Box>
-            <Text ml={2}>Collections</Text>
-          </MinterButton>
-        </Flex>
-        <Flex align="center" justify="center" flex="1" px={16}>
-          <AspectRatio
-            ratio={1}
-            width="100%"
-            borderRadius="3px"
-            boxShadow="0 0 5px rgba(0,0,0,.15)"
-            overflow="hidden"
-          >
-            <Box>
-              <TokenImage
-                src={ipfsUriToGatewayUrl(
-                  system.config.network,
-                  token.artifactUri
-                )}
+    <Flex flexDir="column" bg="brand.brightGray">
+      <Modal
+        isOpen={isOpen}
+        onClose={onClose}
+        size="full"
+        scrollBehavior="inside"
+      >
+        <ModalOverlay />
+        <ModalContent
+          overflow="auto"
+          m="1rem"
+          maxHeight="calc(100vh - 2rem)"
+          display="unset"
+        >
+          <ModalCloseButton left="0.75rem" />
+          <TokenImage
```

Comment:
This would probably look better centered and without the large empty white background:

Reply:
![image](https://user-images.githubusercontent.com/703880/112412650-df369980-8cec-11eb-9522-6d25e5b24463.png)

Reply:
Also, cannot close a video file (clicking just start/stops the player)

Reply:
Large image is cropped weird:
![image](https://user-images.githubusercontent.com/703880/112413467-3f7a0b00-8cee-11eb-9522-1725baa07132.png)